scss/modules/AI-ALLY.scss

```diff
@@ -0,0 +1,170 @@
+header {
+  background-color: rgb(255, 255, 255);
+  width: 100vw;
+  height: 5.5em;
+  border-bottom: 1px solid rgb(236, 240, 241);
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
+  position: fixed;
+  z-index: 5000;
+
+  .row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 75rem;
+    top: 1rem;
+    position: relative;
+    left: 1rem;
+
+    .center-header {
+      ul {
+        display: flex;
+        gap: 20px;
+        padding: 0px;
+        margin: 0px;
+        li a:hover {
+          background-color: none;
+          color: $GREEN-BUTTON-BASKET;
+        }
+      }
+    }
+
+    .left-haeder {
+      display: flex;
+
+      img {
+        position: relative !important;
+      }
+    }
+  }
+
+  @include mobile {
+    .row {
+      flex-direction: column;
+      align-items: center;
+
+      .center-header ul {
+        flex-direction: column;
+        text-align: center;
+      }
+    }
+  }
+
+  @include tablet {
+    .row {
+      .center-header ul {
+        gap: 10px;
+      }
+
+      .right-header {
+        button {
+          padding: 0.4rem 0.8rem;
+        }
+      }
+    }
+  }
+
+  @include laptop {
+    .row {
+      width: 100%;
+      .center-header ul {
+        gap: 15px;
+      }
+    }
+  }
+}
+
+.heroSection {
```

Comment:
why you have different style of class naming?! 
somewhat you define a class name in camleCase and sometimes CamleCase and sometimes with dash - ,.... It's better to have the same structure of the class name for all the classes